erpnext/manufacturing/doctype/production_order/production_order.py

```diff
@@ -634,3 +634,9 @@ def stop_unstop(production_order, status):
 	pro_order.notify_update()
 
 	return pro_order.status
+
+@frappe.whitelist()
+def query_sales_order(production_item):
+	out = frappe.db.sql_list("select distinct so.name from `tabSales Order` so, `tabSales Order Item` so_item\
```

Comment:
Consider both `Sales Order Item` and `Packed Item` table for matching items to find out sales orders.